src/Blaster.WebApi/Features/Capabilities/capabilityservice.js

```diff
@@ -11,8 +11,8 @@ export default class CapabilityService {
     }
 
     getAll() {
-        return this.client.get(this.baseUrl)
-            .then(data => data.items || []);
+        var x = this.client.get(this.baseUrl);

```

Comment:
Use descriptive variable names. Even thou the X-men are kind cool ;)

Reply:
Missed it, thanks!